custom/icds_reports/migrations/0102_drop_update_aggregate_awc_data.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0101_add_supervisor_id'),
+    ]
+
+    operations = [
+        migrations.RunSQL('DROP FUNCTION IF EXISTS update_aggregate_awc_data(date)')
```

Comment:
it looks like there's also a reference to the SQL file in the tasks.py that you'll need to remove to get tests to pass

Reply:
36e1887c303